daemon/mgr/container.go

```diff
@@ -110,11 +110,9 @@ func (mgr *ContainerManager) Restore(ctx context.Context) error {
 		mgr.NameToID.Put(c.Name, c.ID)
 
 		// put container into cache.
-		mgr.cache.Put(c.ID, &Container{
-			meta: c,
-		})
+		mgr.cache.Put(c.ID, c)
```

Comment:
c is a CotnainerMeta, the cache is used to store "Container".

Reply:
Updated. PTAL @skoo87